README.md

```diff
@@ -18,7 +18,8 @@ mack.type_2_scd_upsert(path, updatesDF, "pkey", ["attr1", "attr2"])
 
 ## Type 2 SCD Upserts
 
-This library provides an opinionated, conventions over configuration, approach to Type 2 SCD management.  Let's look at an example before covering the conventions required to take advantage of the functionality.
+This library provides an opinionated, conventions over configuration, approach to Type 2 SCD management. Let's look at an example before
+covering the conventions required to take advantage of the functionality.
```

Comment:
I prefer using word wrap in the README instead of adding these new lines... is this being caused by the code formatter?  Can we instruct it to ignore Markdown files?

Reply:
Will take a look into it 👍